agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/CodyContextFilterItem.kt

```diff
@@ -1,8 +1,15 @@
 @file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
 package com.sourcegraph.cody.agent.protocol_generated;
 
+import com.google.gson.annotations.SerializedName;
+
 data class CodyContextFilterItem(
-  val repoNamePattern: String,
+  val repoNamePattern: RepoNamePatternEnum, // Oneof: .*
   val filePathPatterns: List<String>? = null,
-)
+) {
+
+  enum class RepoNamePatternEnum {
+    @SerializedName(".*") ``,
```

Comment:
```suggestion
    @SerializedName(".*") MATCH_ALL,
```

Change the empty backticks they cause this issue 
```

> Task :compileKotlin FAILED
e: file:///Users/arafatkhan/Desktop/cody/jetbrains/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/CodyContextFilterItem.kt:16:35 Syntax error: Expecting ';' after the last enum entry or '}' to close enum class body.
e: file:///Users/arafatkhan/Desktop/cody/jetbrains/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/CodyContextFilterItem.kt:17:28 Syntax error: Expecting member declaration.
e: file:///Users/arafatkhan/Desktop/cody/jetbrains/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/CodyContextFilterItem.kt:17:29 Syntax error: Expecting member declaration.
e: file:///Users/arafatkhan/Desktop/cody/jetbrains/src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/Constants.kt:10:14 Syntax error: Expecting property name or receiver type.

FAILURE: Build failed with an exception.
```